pages/garden.js

```diff
@@ -0,0 +1,7 @@
+import GardenPage from "@/components/GardenPage/GardenPage";
+
+export default function Garden({ petCollection, onInteractPet }) {
+  return (
+    <GardenPage petCollection={petCollection} onInteractPet={onInteractPet} />
+  );
+}
```

Comment:
I wouldn't use a separate `GardenPage` component here since you're only returning one component. This adds unnecessary complexity. 

Also, the name "GardenPage" is a bit confusing since it's not actually a full page component.